guides/plugins/plugins/framework/data-handling/add-data-translations.md

```diff
@@ -26,9 +26,20 @@ In this guide we'll name our table `swag_example_translation` since our entity i
 
 The translation table's columns should be the following:
 
-swag\_example\_idThis will refer to the swag\_example this translation belongs to. This is also a foreign key.language\_idThis will contain the ID of the language for this translation. This is also a foreign key.nameThe actual translated value, the translated name of the swag\_example.created\_atNot much explanation required here.updated\_atNot much explanation required here.
```

Comment:
I really wonder why the linter from GitBook does this.

Thank you for fixing it!

Reply:
it's broken again … I think it removes HTML completely … can you advise how to proceed here?

Reply:
<img width="785" alt="image" src="https://user-images.githubusercontent.com/930706/126605586-1427b267-1e85-4e5d-9518-680f6f5ff6e5.png">

Reply:
@PaddyS  *ping* (keine ahnung, ob sich github bei dir gemeldet hatte)

Reply:
I haven't really found a solution for this.
I already talked with @elkmod about this issue, because it keeps messing up stuff over and over again.

IIRC, we can't really disable the Linter from GitBook, even though we'd love too.